Dockerfile

```diff
@@ -7,8 +7,8 @@ COPY .env /code/.env
 COPY manage.py /code/manage.py
 COPY jarbas /code/jarbas
 WORKDIR /code
-RUN python manage.py migrate
-RUN python manage.py ceapdatasets
+#RUN python manage.py migrate
+#RUN python manage.py ceapdatasets
```

Comment:
If that's the way, can we delete these lines instead of commenting them? I feel like leaving them commented may cause some confusion.